mcstatus/bedrock_status.py

```diff
@@ -0,0 +1,80 @@
+from time import perf_counter
+import asyncio_dgram
+import socket
+import struct
+
+
+class BedrockServerStatus:
```

Comment:
I don't think it's similar enough, mainly due to that one uses TCP and the other uses UDP. However for consistency and clarity we could rename the file to `bedrock_pinger.py` and `BedrockServerStatus` to `BedrockServerPinger`?

Reply:
Maybe I'm splitting hairs here since these are basically internal classes that are not exported at the top-level `__init__.py` file. On further thinking about this, I'm okay with it as is.